larq_compute_engine/tflite/python/interpreter.py

```diff
@@ -40,11 +42,17 @@ def __init__(
         flatbuffer_model: bytes,
         num_threads: int = 1,
         use_reference_bconv: bool = False,
+        use_indirect_bgemm: bool = False,
+        use_xnnpack: bool = False,
```

Comment:
I think this changes the default or am I missing something? I think that's fine, but can you explain the reasoning behind it?

Reply:
hi @lgeiger. following the discussion on #713 I've implemented it like this.
A) one reason to include this flag in the *LceInterpreter* is to make it reflect more the behavior of the benchmark binary.
B) the second reason is dynamic allocation, thus having models with `allocation_type==kTfLiteDynamic`.
we had models like this and when we set `use_xnnpack=true` in the *benchmark_model* there will be an error thrown. this would be true for *LCEInterpreter* as well, as under the hood XNNPACK is "silently" applied.